schemas/computation.schema.tpl.json

```diff
@@ -0,0 +1,84 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/Computation",
+    "_extends": "/core/research/activity.schema.tpl.json",
+    "required": [
+        "input",
+        "output",
+        "protocol",
+        "environment",
+        "startedAtTime"
+    ],
+    "properties": {
+        "input": {
+            "_linkedTypes": [
+                "https://openminds.ebrains.eu/core/FileBundle",
+                "https://openminds.ebrains.eu/core/FileInstance",
+                "https://openminds.ebrains.eu/products/SoftwareVersion"
```

Comment:
How can SoftwareVersion be an input? Is a SoftwareVersion actually manipulated by the Computation to produce a different SoftwareVersion as output or is it used by the Computation to produce an output?

Reply:
This would be the user's own code, typically, e.g. the top-level Python script. 

You could also include all dependencies (e.g. NumPy, ...), but usually these would be considered part of the Environment. If you installed a specific version of a dependency that is different to the version in the environment, it could be added here.